doc/deps.md

```diff
@@ -91,31 +94,33 @@ other commands below, unless *explicitly* specified!**
 ### Thrift
 CodeCompass needs [Thrift](https://thrift.apache.org/) which provides Remote
 Procedure Call (RPC) between the server and the client. Thrift is not part of
-the official Ubuntu 16.04 LTS and 18.04 LTS repositories, but you can download it
-and build from source:
+the official Ubuntu 16.04 LTS and 18.04 LTS repositories, but you can download
+it and build from source:
 
 ```bash
-# Thrift may require yacc and flex as dependency:
-sudo apt-get install byacc flex
```

Comment:
Are these dependencies really not required? Then the CI shall be modified accordingly, too!

Reply:
I tried it in the Docker image and it works without these.

Reply:
The official [requirements list](http://thrift.apache.org/docs/install/) for Thrift still lists them as mandatory dependencies. Isn't it possible that they were simply installed as a dependency of another package?